security/vex/fleet/CVE-2025-46569.vex.json

```diff
@@ -14,6 +14,9 @@
         {
           "@id": "fleet"
         },
+        {
+          "@id": "pkg:golang/github.com/open-policy-agent/opa@v0.44.0"
```

Comment:
`trivy` requires `v` prefix in the version, `docker scout` does not, thus we need to define them both in the `products` field.